odp_importer/assignlicense4org.py

```diff
@@ -0,0 +1,49 @@
+
+import argparse
+import ckanapi
+
+from ckan_util import check_endpoints, logger
+
+
+def assign_license(ckan_api, org, license):
+    """Set the license for all datasets owned by an org to the one given"""
+
+    license_id = license['id']
+    license_title = license['title']
+    for dataset in ckan_api.action.organization_show(id=org)['packages']:
+        package = ckan_api.action.package_show(id=dataset['id'])
+        package['license_id'] = license_id
+        package['license_title'] = license_title
+        logger.info('Set license on %s to %s' % (package['title'],
+                                                 license_title))
+        ckan_api.action.package_update(**package)
+
+
+def get_license(ckan_api, license_id):
+    """Pick out a license dict for a given license id or title"""
+
+    for license in ckan_api.action.license_list():
+        if license['id'] == license_id or license['title'] == license_id:
+            return license
+    return None
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Set a license for all '
+                                                 'datasets owned by an '
+                                                 'organization')
+    parser.add_argument('org', metavar='ORG', help='Organization')
```

Comment:
It'd be good to expand the help text to be clear that this is the `name` field (I think that's the CKAN term for the slug) for the organization. Same for the licenses.